sktm/__init__.py

```diff
@@ -206,16 +206,18 @@ def check_patchwork(self):
                                              pids)
                 # Submit and remember a Jenkins build for the patchset
                 self.pj.append((sktm.jtype.PATCHWORK,
-                                self.jk.build(self.jobname,
-                                              baserepo=self.baserepo,
-                                              ref=stablecommit,
-                                              baseconfig=self.cfgurl,
-                                              patchwork=patchset,
-                                              emails=emails,
-                                              makeopts=self.makeopts),
+                                self.jk.build(
+                                    self.jobname,
+                                    baserepo=self.baserepo,
+                                    ref=stablecommit,
+                                    baseconfig=self.cfgurl,
+                                    message_id=patchset.message_id,
+                                    emails=patchset.email_addr_set,
+                                    patchwork=patchset.patch_url_list,
+                                    makeopts=self.makeopts),
                                 cpw))
-                logging.info("submitted patchset: %s", patchset)
-                logging.debug("emails: %s", emails)
+                logging.debug("emails: %s", patchset.email_addr_set)
+                logging.info("submitted patchset: %s", patchset.patch_url_list)
```

Comment:
The PatchsetSummary class isn't logging anything so I fail to understand how is that related. It would be OK if we were logging the identifying Message-ID here, but since we are not it's confusing. I'd say either keep the original order since the patch URL kind of serves as identifier, or log the Message-ID before the emails too.

Point is that with logging, I think it's important to keep some kind of `identifier: related logs` order which is missing here, if that makes sense.

Reply:
Well, true, it's not logging anything. What I'm aiming at here is having a consistent data hierarchy and order across everything. I agree with the `identifier: related` order. I'm adding `Message-ID` here too. I think I just forgot to do that in a hurry.